### PR TITLE
fix(helm): Moved priority class declaration in the right position of dapr_placement_statefulset.yaml manifests

### PR DESCRIPTION
**Pull request by m3nax**
# Description

Dapr, when deployed in HA mode with a priorityClassName defined as suggested in this [doc](https://docs.dapr.io/operations/hosting/kubernetes/kubernetes-production/#high-availability-mode), produce an invalid manifeset.
It add the **priorityClassName** inside the **volumeClaimTemplates** definition of **dapr-placement-server** `StatefulSet`.

Command used to produce the manifest
``` BASH
helm template .\dapr\ --set global.ha.enabled=true --set global.priorityClassName=system-cluster-critical > example-manifest-dapr.yaml
```
Produced **volumeClaimTemplate** of **dapr-placement-server** `StatetfulSet`:
``` YAML
  volumeClaimTemplates:
  - apiVersion: v1
    kind: PersistentVolumeClaim
    metadata:
      name: raft-log
    spec:
      accessModes:
        - ReadWriteOnce
      resources:
        requests:
          storage: 1Gi
      priorityClassName:
        system-cluster-critical
```

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

- [x] Code compiles correctly
- [ ] Created/updated tests
- [ ] Unit tests passing
- [ ] End-to-end tests passing
- [ ] Extended the documentation / Created issue in the <https://github.com/dapr/docs/> repo: dapr/docs#_[issue number]_
- [ ] Specification has been updated / Created issue in the <https://github.com/dapr/docs/> repo: dapr/docs#_[issue number]_
- [ ] Provided sample for the feature / Created issue in the <https://github.com/dapr/docs/> repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Review by elena-kolevska — APPROVED** [review]
Thanks for catching this @m3nax !

**Comment by m3nax** [comment]
@elena-kolevska Is there anything else I can do on this pull request to make it merge?

**Comment by elena-kolevska** [comment]
I don't think so, we now have to wait for one of the maintainers to approve and merge.

File: charts/dapr/charts/dapr_placement/templates/dapr_placement_statefulset.yaml
Comment (cicoyle) [inline charts/dapr/charts/dapr_placement/templates/dapr_placement_statefulset.yaml:256]: Thanks for finding this - mind adding this same diff for this file: `dapr_scheduler_statefulset.yaml`?

Comment (m3nax) [reply charts/dapr/charts/dapr_placement/templates/dapr_placement_statefulset.yaml:256]: It seems to be already present in the file `dapr_scheduler_statefulset.yaml`

https://github.com/m3nax/dapr/blob/24ba30bc4879eee63106aed877858f322f4ca223/charts/dapr/charts/dapr_scheduler/templates/dapr_scheduler_statefulset.yaml#L271-L274

Comment (cicoyle) [reply charts/dapr/charts/dapr_placement/templates/dapr_placement_statefulset.yaml:256]: ah, yes I mixed up the indentation and spec and was thinking we needed this same diff for the scheduler, but we dont. thx, you are correct we are good there.

**Comment by m3nax** [comment]
@cicoyle  I think the errors in the actions are not related to my modification, right?

**Review by cicoyle — APPROVED** [review]

**Pull request merged**
